### DISCUSSION TITLE
chromixs SEC-SAXS elution profile errors

### DISCUSSION THREAD

**Discussion by I-G-E** [biosaxs-com/atsas-community#1948] [Primary data processing] [answered]
Hi,

Apologies if this has been asked before. I am using ATSAS 4.0.1 (thanks for the update to fix chromixs crashing!) and wondering how the errors that are shown for the average intensity on a SEC-SAXS elution profile are calculated and also the errors for the Rg for each frame - are they standard error or standard deviation or calculated in another way? 

Thanks in advance.

**Comment by franke-biosaxs** [comment, marked as answer]
The errors on the trace are propagated from the intensities used to calculate the trace itself, i.e. $$\left(\frac{1}{n}\sum_{s=s_{min}}^{s_{max}} \sigma(I(s))^2\right)^{1/2}$$. If I got the LaTeX right on that one.

The errors on the Rg are the errors from autorg. Which, having said that, I now realise aren't the best values to be used (cf. my answer for #1942).

**Comment by franke-biosaxs** [comment]
Btw, I am currently working an extensive [manual for the new chromixs](https://biosaxs-com.github.io/atsas/4.0.1/manuals/chromixs.html). It is still work in progress, but things are slowly coming together. In case you have a look, I would appreciate any feedback you might have!

**Reply by I-G-E** [reply]
Thanks for the info - I'll take a look! Another thing I wanted to ask about Chromixs - when you select your sample frames and buffer frames and then calculate structural parameters, it only calculates the Rg values for the selected sample frames (rather than for the whole peak). Is there a way to calculate the Rg values for the whole peak as I would like to show the Rg trend across the whole peak as well as for just the selected sample frames.

**Reply by franke-biosaxs** [reply]
That's a new question that doesn't fit the subject - please post it in a separate thread, then I'm happy to answer.
